reference/tools/user-guide.md

```diff
@@ -6,184 +6,270 @@ aliases: ['/docs/dev/how-to/migrate/from-mysql/','/docs/dev/how-to/migrate/incre
+- Supported TiDB versions: all versions
+- Kubernetes support: Yes. See [Backup and Restore](/tidb-in-kubernetes/maintain/backup-and-restore.md) for details.
 
-#### Full export tool TiDB Binlog
+#### BR (beta)
 
-[TiDB Binlog](/reference/tidb-binlog/overview.md) is a tool used to collect binlog data from TiDB. It provides near real-time backup and replication to downstream platforms.
+[BR](/reference/tools/br/br.md) (Backup & Restore) is a command-line tool for distributed backup and restoration of the TiDB cluster data. Compared with Mydumper/loader, BR is more suitable for scenarios of huge data volume.
 
-The following are the basics of TiDB Binlog:
+The following are the basics of BR:
+
+- Input/Output
+    - Input: TiDB cluster
+    - Output: Full backup file
+- Supported TiDB versions: v3.1 or v4.0
+- Kubernetes support: Yes. The document is WIP.
 
-- Input: TiDB clusters.
-- Output: MySQL, TiDB, Kafka or incremental backup files.
-- Compatibility: Compatible with TiDB v2.1 and later versions.
-- Kubernetes: Supported. See [TiDB Binlog Cluster Operations](/tidb-in-kubernetes/maintain/tidb-binlog.md) and [TiDB Binlog Drainer Configurations in Kubernetes](/tidb-in-kubernetes/reference/configuration/tidb-drainer.md).
+### Incremental data export tools
 
-## Tools development roadmap
+#### TiDB Binlog
 
-To help you understand the relationships between the above tools, here is a brief introduction to TiDB ecosystem tools development roadmap.
+[TiDB Binlog](/reference/tidb-binlog/overview.md) is a system that collects binlog for TiDB clusters and provides tools for near real-time sync and backup.
 
-### TiDB backup and restore
+The following are the basics of TiDB Binlog:
 
-Mydumper and Loader -> BR:
+- Input/Output:
+    - Input: TiDB Cluster
+    - Output: MySQL, TiDB, Kafka or incremental backup files
+- Supported TiDB versions: v2.1 or later
+- Kubernetes support: Yes. See [TiDB Binlog Cluster Operations](/tidb-in-kubernetes/maintain/tidb-binlog.md) and [TiDB Binlog Drainer Configurations in Kubernetes](/tidb-in-kubernetes/reference/configuration/tidb-drainer.md) for details.
+
+#### CDC (Beta, under development, ETA May/June 2020 with TiDB 4.0)
+
+[CDC](/reference/tools/ticdc/overview.md) (Change Data Capture) is a system that collects changelog for key value pairs in TiKV and outputs to downstream systems in transactional order.
+
+- Input/Output: 
+    - Input: TiDB Cluster
+    - Output: MySQL, TiDB, Kafka or incremental backup files
+- Supported TiDB versions: v4.0
+- Kubernetes support: On the development road map, ETA Q2 2020
+
+## Recommended tools for TiDB versions
+
+### Recommended tools for TiDB 3.0 or earlier
+
+- MySQL full data backup: use Mydumper
+- MySQL full data import to TiDB:
+    - TB scale: use TiDB Lightning
+    - Sub-TB scale: Use DM
+- MySQL incremental data sync to TiDB: use DM
+- TiDB full data backup: Use Mydumper
+- TiDB full data restore:
+    - TB scale: use TiDB Lightning
+    - Sub-TB scale: Use DM
```

Comment:
dm doesn't support it,  dm only support mysql -> tidb, please remove it. 

@WangXiangUSTC Please take a carful look

Reply:
`- Sub-TB scale: Use DM` is changed to `- Sub-TB scale: Use TiDB Lightning`.